bundle/lib/source/DobbySpecConfig.cpp

```diff
@@ -618,7 +618,8 @@ bool DobbySpecConfig::parseSpec(ctemplate::TemplateDictionary* dictionary,
     // step 6.5 - enable localtime rdk plugin by default if on Xi1. The
     // localtime plugin takes no input params, so we simply enable the
     // rdkPlugin rather than processing it via a processing function.
-    mRdkPluginsJson[RDK_LOCALTIME_PLUGIN_NAME]["data"] = "{}";
+    Json::Value rdkPluginData = Json::objectValue;
+    mRdkPluginsJson[RDK_LOCALTIME_PLUGIN_NAME]["data"] = rdkPluginData;
```

Comment:
LGTM, also I think that just using Json::objectValue should be enough (compare jsoncpp/include/json/value.h line 312), but your solution will always work without doubts.

Reply:
> To be honest I get
> 
> ```
> "localtime":{
>       "required":false,
>       "data":{}
>    }
> ```
> 
> result with my unchanged XiOne. It is two week old build, so was this some kind of regression?

Yes, regression was introduced in https://github.com/rdkcentral/Dobby/pull/54